### PR TITLE
Switch to first/last tab

### LINKED ISSUE #35: Add "I switch to last tab" step
Add "I switch to last tab" step to `When` steps (https://github.com/revjet-qa/wdio-cucumber-steps/blob/master/src/steps/when.js)

### PR DESCRIPTION
**Pull request by robomanus**
This pull request adds 'I switch to first/last tab' step

Fixes #35

### CODE REVIEW

File: test/features/window.tab.manipulations.feature [hunk 1]
```diff
@@ -15,7 +15,18 @@ Feature: Window/tab
     And I close current tab
 
   Scenario: "I close current tab" step should close recently opened tab
-    Given I open "http://localhost:9000"    
+    Given I open "http://localhost:9000"  
+    When I wait "5000" ms
```
Comment (Marketionist) [inline test/features/window.tab.manipulations.feature]: Why do you need to wait for 5 sec here?

File: test/features/window.tab.manipulations.feature [hunk 2]
```diff
@@ -15,7 +15,18 @@ Feature: Window/tab
...
     Then txtHeader from main page should be present
+
+  Scenario: "I switch to first tab" steps should open first tab
+    Given I open "http://localhost:9000"  
+    When I open new window
+    And I switch to first tab
+    Then txtHeader from main page should be present
+  
+  Scenario: "I switch to last tab" steps should open last tab
+    When I switch to last tab
```
Comment (Marketionist) [inline test/features/window.tab.manipulations.feature]: Each test should be independent from the others, so it looks like you have to add
```
Given I open "http://localhost:9000"  
And I open new window
And I switch to first tab
```
at the beginning of this test

File: src/steps/when.js [hunk 3]
```diff
@@ -97,5 +97,37 @@ module.exports = function () {
...
+                return await browser.switchTab(firstTabId);
+            } catch (err) {
+                throw new Error(err);
+            }
+        });
+
+        When(_r('I switch to last tab$'), async function async () {
+            /**
+             * Open last tab
+             * @param {String} err - Any error message
```
Comment (Marketionist) [inline src/steps/when.js]: This function does not have any incoming parameters, so you do not need to describe any `@param`

File: src/steps/when.js [hunk 4]
```diff
@@ -97,5 +97,37 @@ module.exports = function () {
             }
         });
 
+        When(_r('I switch to first tab$'), async function async () {
+            /**
+             * Open last tab
+             * @param {String} err - Any error message
```
Comment (Marketionist) [inline src/steps/when.js]: This function does not have any incoming parameters, so you do not need to describe any `@param`

Comment (robomanus) [reply src/steps/when.js]: Implemented

File: src/steps/when.js (hunk 3 above)
Comment (robomanus) [reply src/steps/when.js]: Implemented

File: test/features/window.tab.manipulations.feature (hunk 2 above)
Comment (robomanus) [reply test/features/window.tab.manipulations.feature]: Implemented

File: test/features/window.tab.manipulations.feature (hunk 1 above)
Comment (robomanus) [reply test/features/window.tab.manipulations.feature]: Forgot to delete after debugging, fixed

**Comment by Marketionist** [comment]
LGTM

**Pull request merged**